source/module_base/parallel_global.cpp

```diff
@@ -172,43 +172,56 @@ void Parallel_Global::read_mpi_parameters(int argc,char **argv)
 
     //  GlobalV::KPAR = atoi(argv[1]); // mohan abandon 2010-06-09
 
-	// get the size --> GlobalV::NPROC
-	// get the rank --> GlobalV::MY_RANK
-	MPI_Comm_size(MPI_COMM_WORLD,&GlobalV::NPROC);
+    // get world size --> GlobalV::NPROC
```

Comment:
Could you use the format of Doxygen? https://mcresearch.github.io/abacus-user-guide/develop-dox.html

Reply:
This is a part of comments, and doxygen will not generate documentation for this part. It's adequate to add the comment to the function declaration in the header file, i.e. global_variable.h.